shell/utils/ui-plugins.ts

```diff
@@ -0,0 +1,263 @@
+import { CATALOG as CATALOG_ANNOTATIONS } from '@shell/config/labels-annotations';
```

Comment:
A bit of a nitpick, but could you not hyphenate the filename? everything in dashboard is `uiplugins` and not `ui-plugins` 😬  🙏

Reply:
Overall, this is a really nice addition mate! Thanks!